src/demo.js

```diff
@@ -208,24 +193,31 @@ export default class Demo extends Component {
                                 step="1"
                                 onChange={evt => this.handleExternalsChange(evt.target.name, parseInt(evt.target.value, 10))}
                             />
-                            <input
-                                className="externals__input"
-                                type="number"
-                                name="photo_count"
-                                value={externals.photo_count}
-                                onChange={evt => this.handleExternalsChange(evt.target.name, parseInt(evt.target.value, 10))}
-                            />
```

Comment:
Removing this doesn't allow the user to test what happens when there are 1000 or 10000 photos. Is there a way to show case bug numbers as well as small ones?